packages/react-components/react-jsx-runtime/README.md

```diff
@@ -1,5 +1,49 @@
 # @fluentui/react-jsx-runtime
 
-**React Jsx Runtime components for [Fluent UI React](https://react.fluentui.dev/)**
+**React JSX runtime for [Fluent UI React](https://react.fluentui.dev/)**
 
-These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+[Fluent UI React](https://react.fluentui.dev/) requires the usage of a custom JSX runtime to support the [slots API](https://react.fluentui.dev/?path=/docs/concepts-developer-customizing-components-with-slots--page)
+
+## Usage
+
+This library should only be used in the case where you are trying to use the internal slot API of Fluent UI React. If you are not using the internal slot API, you should not need to use this library.
+
+In case you want to re-compose a component and redeclare its render method then this API will be necessary, here's our documentation on [Rendering a component with slots](https://react.fluentui.dev/?path=/docs/concepts-developer-customizing-components-with-slots--page#rendering-components-with-slots)
+
+To properly render a component with slots the `createElement` method of `@fluentui/react-jsx-runtime` can be used as a replacement for `React.createElement`:
+
+```tsx
+/** @jsxRuntime classic */
+/** @jsx createElement */
+
+// createElement custom JSX pragma is required to support slot creation
+import { createElement } from '@fluentui/react-jsx-runtime';
```

Comment:
```suggestion
/** @jsxRuntime automatic */
/** @jsxImportSource @fluentui/react-jsx-runtime */
```

Why not suggest usage of new pragma?

Reply:
Because we're not using it ourselves yet?! 😅
As soon as we start using I'll update this to suggest usage of new pragma